### ISSUE TITLE
[Core] Support "include" Argument for Ray Runtime Environment

### ISSUE THREAD

**Issue by MengjinYan** [ray-project/ray#49136]
### Description

Today, Ray's runtime environment supports "exclude" argument to exclude files when creating the runtime environment. The ask is to also support "include" to only include certain files to the runtime environment.

### Use case

For the use cases where only a small amount of files under a large directory is needed for the runtime environment

**Comment by Jay-ju** [comment]
I want to try to solve this problem. I think it's quite useful. Is anyone else dealing with this problem now?

**Comment by MengjinYan** [comment]
That sounds great!! I don't think anyone is working on the issue right now. Please feel free to send us PRs or ask any questions that you might have. :D cc: @jjyao @dentiny

**Comment by dentiny** [comment]
Looking forward to your PR! @Jay-ju

**Comment by ArturNiederfahrenhorst** [comment]
@Jay-ju Any update on your plans to tackle this?

**Comment by ruisearch42** [comment]
Added back triage label so that we will plan for this task.

**Comment by fscnick** [comment]
Hi, I'd like to try to solve this issue.

**Comment by MortalHappiness** [comment]
Removed triage label because there is a new assignee.

**Comment by Jay-ju** [comment]
> [@Jay-ju](https://github.com/Jay-ju) Any update on your plans to tackle this?


yes, it is already being processed. It will be brought up later.

**Comment by Jay-ju** [comment]
> Hi, I'd like to try to solve this issue.

Wow! Thank you so much!
 However, I am already handling it. This issue is quite useful in our scenario. I will submit a pull request later.

**Comment by fscnick** [comment]
> > Hi, I'd like to try to solve this issue.
> 
> Wow! Thank you so much! However, I am already handling it. This issue is quite useful in our scenario. I will submit a pull request later.

Thank for the reply. I'll put it on hold.
